From bf8b46717f3068e01dab232f89facc184f5772e3 Mon Sep 17 00:00:00 2001
From: Jon West <electrikjesus@gmail.com>
Date: Thu, 27 Jun 2024 11:47:52 -0400
Subject: [PATCH 18/23] fix: adjust watchdog timeout for long tasks

This adds ro.hw_timeout_multiplier property to the device
---
 device.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/device.mk b/device.mk
index eee14de..5e375e5 100644
--- a/device.mk
+++ b/device.mk
@@ -74,6 +74,10 @@ endif
 PRODUCT_PRODUCT_PROPERTIES += \
     persist.sys.fuse.passthrough.enable=true
 
+# Watchdog
+PRODUCT_PROPERTY_OVERRIDES += \
+    ro.hw_timeout_multiplier=3
+
 PRODUCT_COPY_FILES := \
     $(if $(wildcard $(PRODUCT_DIR)init.rc),$(PRODUCT_DIR)init.rc:root/init.rc) \
     $(if $(wildcard $(PRODUCT_DIR)init.sh),$(PRODUCT_DIR),$(LOCAL_PATH)/)init.sh:system/etc/init.sh \
-- 
2.34.1

